Program.cs

```diff
@@ -1,25 +1,41 @@
 ﻿const string filePath = "input.txt";
-
+int letter_count = 0;
+int DigitCount = 0;
 try
 {
-    if (File.Exists(filePath))
+    using var reader = new StreamReader(filePath);
+
+    while (!reader.EndOfStream)
     {
-        var content = File.ReadAllText(filePath);
 
-        int letterCount = content.Count(char.IsLetter);
```

Comment:
It would have been enough to change these IsLetter, IsDigit methods with your new specific count methods from Helper class and not to have all this loop